jcl/src/java.base/share/classes/jdk/crac/Core.java

```diff
@@ -69,7 +74,12 @@ class CRIUSupportContext<R extends Resource> extends Context<R> {
 	// InternalCRIUSupport.getCRaCCheckpointToDir() is not null if
 	// InternalCRIUSupport.isCRaCSupportEnabled() returns true before creating CRIUSupportContext<>().
 	private final InternalCRIUSupport internalCRIUSupport = new InternalCRIUSupport(
-			Paths.get(InternalCRIUSupport.getCRaCCheckpointToDir())).setLeaveRunning(false).setShellJob(true)
+			Paths.get(InternalCRIUSupport
+			.getCRaCCheckpointToDir()))
```

Comment:
Suggest keeping `Paths.get(InternalCRIUSupport.getCRaCCheckpointToDir()))` in one line.